src/app/query/stacks/balances/balances.hooks.ts

```diff
@@ -54,29 +58,49 @@ export function useCurrentAccountUnanchoredBalances() {
   return useAddressBalances(account?.address || '');
 }
 
-export function useBaseAssetsUnanchored() {
+export function useStacksNativeAsset() {
   const balances = useCurrentAccountUnanchoredBalances();
   return useMemo(() => {
-    return transformAssets(balances.data);
+    return convertStxBalanceToBaseAssetType(balances.data);
+  }, [balances]);
+}
+
+export function useStacksAssetsFtUnanchored() {
+  const balances = useCurrentAccountUnanchoredBalances();
+  return useMemo(() => {
+    return convertFtBalancesToStacksAssetFtType(balances.data);
+  }, [balances]);
+}
+
+export function useStacksAssetsNftUnanchored() {
+  const balances = useCurrentAccountUnanchoredBalances();
+  return useMemo(() => {
+    return convertNftBalancesToStacksAssetNftType(balances.data);
   }, [balances]);
 }
 
 function useAddressAnchoredBalances(address: string) {
-  const { data: balances } = useGetAnchoredAccountBalanceQuery(address, {
+  return useGetAnchoredAccountBalanceQuery(address, {
     select: (resp: AddressBalanceResponse) => initAmountsAsMoney(resp),
     retryOnMount: true,
     keepPreviousData: false,
   });
-  return balances;
 }
 
-export function useCurrentAccountAnchoredBalances() {
+function useCurrentAccountAnchoredBalances() {
   const account = useCurrentAccount();
   return useAddressAnchoredBalances(account?.address || '');
 }
 
-export function useAddressAnchoredAvailableStxBalance(address: string) {
-  const balances = useAddressAnchoredBalances(address);
+export function useStacksAssetsFtAnchored() {
+  const balances = useCurrentAccountAnchoredBalances();
+  return useMemo(() => {
+    return convertFtBalancesToStacksAssetFtType(balances.data);
+  }, [balances]);
```

Comment:
Nit, implicit return syntax

```suggestion
  return useMemo(() => convertFtBalancesToStacksAssetFtType(balances.data), [balances]);
```